internal/cmd/user/user.go

```diff
@@ -24,30 +24,38 @@ type CreateParams struct {
 
 var ErrMustSpecifyUser = errors.New("must specify user ID or user name")
 
-func (c Client) Create(ctx context.Context, params *CreateParams) error {
+func getOrgID(ctx context.Context, params *cmd.OrgParams, c cmd.CLI, orgApi api.OrganizationsApi) (string, error) {
```

Comment:
Why these changes?

Reply:
Was just reading this file to understand how to deal with orgs, and it was more obvious to read this way. I think it's likely that this function will eventually be useful elsewhere, but haven't found anywhere else to use it yet